test/new-e2e/tests/gpu/gpu_test.go

```diff
@@ -151,3 +151,18 @@ func (v *gpuSuite) TestVectorAddProgramDetected() {
 		}
 	}, 5*time.Minute, 10*time.Second)
 }
+
+func (v *gpuSuite) TestNvmlMetricsPresent() {
+	// Nvml metrics are always being collected
+	v.EventuallyWithT(func(c *assert.CollectT) {
+		// Not all NVML metrics are supported in all devices. We check for some basic ones
+		metricNames := []string{"gpu.temperature", "gpu.pci.throughput.tx", "gpu.power.usage"}
```

Comment:
so those are ensured to exist on any nvidia device? (to avoid unnecessary flakiness)

Reply:
I couldn't find any that were "ensured", but these seem to be common enough and also work for the instance type we're testing on. The assigned device type to these instances is fixed so we shouldn't have flakiness due to that.